blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -1015,20 +1025,33 @@ async function getTaskStatusMapping(taskStatus) {
 }
 
 async function addThumbnailToItem(itemEl, programName, campaignName, deliverableType) {
-    try {
-        const imageObject = await searchAsset(programName, campaignName,deliverableType);
-        if (imageObject && imageObject.imageUrl) {
-            const thumbnailDiv = itemEl.querySelector('.thumbnail');
-            const imgElement = document.createElement('img');
-            imgElement.src = imageObject.imageUrl;
-            imgElement.alt = imageObject.imageAltText;
-            imgElement.loading = 'lazy';
-            thumbnailDiv.appendChild(imgElement);
-        } else {
-            console.error("Image Object does not have a valid imageUrl");
+    // Create a unique key for the cache based on the parameters
+    const cacheKey = `${programName}-${campaignName}-${deliverableType}`;
```

Comment:
@TyroneAEM can we only add `campaignName` in the `cacheKey` when **campaignName** is not **null** or **empty**